mungegithub/mungers/publish_scripts/publish_template.sh

```diff
@@ -60,10 +60,12 @@ SCRIPT_DIR=$(dirname "${BASH_SOURCE}")
 source "${SCRIPT_DIR}"/util.sh
 
 git fetch origin
+git rebase --abort || true
+git reset --hard
+git clean -f -f -f
```

Comment:
`git clean -ffdx`?

Reply:
What does the third `-f` do? The man page doesn't say.

rmmh's suggestion looks better.